fe1-web/features/rollCall/components/EventRollCall.tsx

```diff
@@ -5,10 +5,11 @@ import { useSelector } from 'react-redux';
 import { useNavigation } from '@react-navigation/native';
 
 import { makeCurrentLao } from 'store';
-import { Timestamp, Wallet } from 'model/objects';
+import { Timestamp } from 'model/objects';
 import QRCode from 'components/QRCode';
 import WideButtonView from 'components/WideButtonView';
 import STRINGS from 'res/strings';
+import * as Wallet from 'features/wallet/objects';
```

Comment:
it seems to me that in the future (not this PR), we could simply export from the wallet two custom hooks, along the lines of `useCurrentPopToken(): PopToken | undefined` and a `usePopTokenGenerator(laoId, rollCallId): PopToken | undefined` to cover virtually all the needs of import the wallet 😄 

Maybe we wouldn't even need to export `usePopTokenGenerator` and we could instead have a component which simply displays the QR code for a generated token, and it'd be enough. Just thinking ... 🤔